docs/markdown/recursor/settings.md

```diff
@@ -367,6 +367,18 @@ Some DNS errors occur rather frequently and are no cause for alarm.
 If set to a digit, logging is performed under this LOCAL facility. See
 [Logging](../common/logging.md#logging). Do not pass names like 'local0'!
 
+## `lowercase-outgoing`
+* Boolean
+* Default: no
+* Available since: 4.0.0
+
+Set to true to lowercase the outgoing queries. When set to 'no' (the default) a
+query from a client using mixed case in the DNS labels ([draft-vixie-dnsext-dns0x20-00](http://tools.ietf.org/html/draft-vixie-dnsext-dns0x20-00)),
```

Comment:
Maybe something like "such as user-entered mixed case names or [draft-vixie-dnsext-dns0x20-00](http://tools.ietf.org/html/draft-vixie-dnsext-dns0x20-00)", to avoid the implication that it's specifically related to the latter?